### DISCUSSION TITLE
Any plans to port this theme to vsCode or neovim ?

### DISCUSSION THREAD

**Discussion by Rem0ld** [AnubisNekhet/AnuPpuccin#105] [Ideas]
Hello,
Thanks for the theme, it's super nice!
I was wondering if you have any plans on porting this theme for vsCode or neovim? 
It would be very much appreciated 🙂

**Comment by AnubisNekhet** [comment]
Hi, most of the colorschemes in this theme are available in VSCode and neovim (As is the case with [nord](https://github.com/gbprod/nord.nvim), [catppuccin](https://github.com/catppuccin/nvim) and [dracula](https://github.com/Mofiqul/dracula.nvim)). Maintaining this visual style across different platforms can be quite challenging and time consuming. I'd suggest using the aforementioned themes on other platforms since they have better attention to detail and support than I can provide by myself! 😁

**Reply by Rem0ld** [reply]
Yes I understand, thanks for this theme anyway I'm using it and loving it
